content/events/pl-colo-cowork.md

```diff
@@ -0,0 +1,35 @@
+---
+eventName: "PL Colo(rado) Cowork"
+
+eventDescription: "A hub for connecting with fellow builders who, like you, are passionate about bringing ideas to life. PL is getting ready to unveil a pop-up co-working space, paired with discounted hotel rooms that you can self-book. For those keen on joining, explore the array of accommodation options available and complete the form linked below." 
+
+eventTopic: "Co-working" 
+
+eventType: "Conference" 
+
+location: "Breckenridge, United States"
+
+venueName: 
+
+venueAddress: 
+
+venueMapsLink: 
+
+website: "https://docs.google.com/forms/d/e/1FAIpQLScgLrRm0MdbogxVFBZwxEJzHPpt_wHXYAE9ghQiQ3edEE8oKQ/viewform" 
+
+startDate: "03/04/2024"
+
+endDate: "03/08/2024" 
+
+tag: "PLN Event" 
+
+dateTBD: false
+
+preferredContacts:
+  - 'form|https://docs.google.com/forms/d/e/1FAIpQLScgLrRm0MdbogxVFBZwxEJzHPpt_wHXYAE9ghQiQ3edEE8oKQ/viewform'
```

Comment:
We currently only support the following means as preferredContacts - Twitter, LinkedIn, Discord and email. Please share details relating to one of the suggested means.

Reply:
@WinsCodes I updated the contact info and the website.